src/docs/contributing/pages/components.mdx

```diff
@@ -99,8 +108,14 @@ Something important, but maybe not _that_ important.
 </Note>
 ```
 
+Use this for these types of content:
+
+- Prerequisite(s): Use these to list things that are required to finish. Place them prior to the procedure/process that follows so a user doesn't start and stop.
+- Note: to document information that's relevant to the topic, but isn't part of the larger point of the content. This might include a piece of information that should be highlighted, but isn't necessarily more important than other information. These include calling out early adopter features or providing clarification. Use sparingly.
```

Comment:
```suggestion
- **Note:** to document information that's relevant to the topic, but isn't part of the larger point of the content. This might include a piece of information that should be highlighted, but isn't necessarily more important than other information. These include calling out early adopter features or providing clarification. Use sparingly.
```